test/rulesets/Base/fastmath_able.jl

```diff
@@ -91,12 +91,9 @@ const FASTABLE_AST = quote
     end
 
     @testset "rounding" begin
-        for x in (-0.1, 6.4, 0.5 + 0.25im)
-            test_scalar(round, x)
-            if x isa Real
-                test_scalar(floor, x)
-                test_scalar(ceil, x)
-            end
+        for x in (-0.6, -0.2, -0.1, 0.1, 0.2, 0.6, 6.4)
+            test_scalar(floor, x, fdm = backward_fdm(5, 1))
+            test_scalar(ceil, x, fdm = forward_fdm(5, 1))
```

Comment:
keyword arguments in BlueStyle are sperated by `;` and don't use spaces around `=`
```suggestion
            test_scalar(floor, x; fdm=backward_fdm(5, 1))
            test_scalar(ceil, x; fdm=forward_fdm(5, 1))
```
https://github.com/invenia/BlueStyle#keyword-arguments

Also it it intentional that the tests on `round` were deleted?
Maybe there are to be moved down to there own section so we don't needthe `if` by you forgot to put them back?

Also do we need to test both `-0.1`, and `-0.2` (similar for positive).
My instinct is we want to test one point on each side of 0.5, and maybe again for negative